.github/workflows/deploy-release.yml

```diff
@@ -0,0 +1,45 @@
+# When a tag is created, create a release build and upload it to Google Play
+
+name: Deploy release to Google Play
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ./.github/workflows/check-and-build.yml
+
+      - name: Build Blue aab
+        run: ./gradlew app:bundleBlueRelease
+
+      - uses: r0adkll/sign-android-release@f30bdd30588842ac76044ecdbd4b6d0e3e813478
+        name: Sign Tusky Blue aab
+        id: sign_aab
+        with:
+          releaseDirectory: app/build/outputs/bundle/blueRelease
+          signingKeyBase64: ${{ secrets.KEYSTORE }}
+          alias: ${{ secrets.KEY_ALIAS }}
+          keyStorePassword: ${{ secrets.KEYSTORE_PASSWORD }}
+          keyPassword: ${{ secrets.KEY_PASSWORD }}
```

Comment:
We can replace this step with:

https://github.com/LawnchairLauncher/lawnchair/blob/2d86033f7a1f0e3fd32ba395118d80e73832c2d3/.github/workflows/ci.yml#L44-L53

https://github.com/LawnchairLauncher/lawnchair/blob/2d86033f7a1f0e3fd32ba395118d80e73832c2d3/build.gradle#L194-L210

Reply:
I'd prefer it if we don't touch the build.gradle files

Reply:
I'll address that in a separated PR.